src/flb_http_client.c

```diff
@@ -531,7 +531,7 @@ struct flb_http_client *flb_http_client(struct flb_upstream_conn *u_conn,
     char *fmt_plain =                           \
         "%s %s HTTP/1.%i\r\n";
     char *fmt_proxy =                           \
-        "%s http://%s:%i/%s HTTP/1.%i\r\n"
```

Comment:
URIs always start with a ```/```, why adding this new one here ?

Reply:
In fact, I’m not adding a `/`, here. I’m removing it from `fmt_proxy` because `uri` is always prefixed by a `/`.
I think I got an issue because of the two consecutive slashes.